### PR TITLE
Implement homework #4 by jsakhno

### PR DESCRIPTION
**Pull request by jsakhno**
Add file "test.py" where we have dict with letters as keys and numbers as value,
class "Test" wich loops printing value from dict and we're parsing letter wich you must inpute in terminal.
Inpute "python tests.py --letter a"("a" is letter wich we parse)in terminal for run.
	новий файл:    homeworks/yaroslav_sakhno/hw04/tests.py

### CODE REVIEW

**Comment by jsakhno** [comment]
https://github.com/Lv-362TAQC/pyatsTraining/pull/20#pullrequestreview-196417722
@ProffesorRabbit fixed

File: homeworks/yaroslav_sakhno/hw04/tests.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import argparse
+from pyats.aetest import Testcase, test, main, loop
+
+mapping = {
+    'a': (1, 3, 4, 5, 6, 7, 8),
+    'b': (0, 2, 3, 4),
+    'c': (7, 9, 0, 6, 5, 4, 3, 1)
+}
+
+parser = argparse.ArgumentParser(description="Parsing letter for mapping")
```
Comment (extsoft) [inline homeworks/yaroslav_sakhno/hw04/tests.py]: @jsakhno please move to main section.

File: homeworks/yaroslav_sakhno/hw04/tests.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+from pyats.aetest import Testcase, test, main, loop
+
+mapping = {
+    'a': (1, 3, 4, 5, 6, 7, 8),
+    'b': (0, 2, 3, 4),
+    'c': (7, 9, 0, 6, 5, 4, 3, 1)
+}
+
+
+def main1():
```
Comment (ProffesorRabbit) [inline homeworks/yaroslav_sakhno/hw04/tests.py]: Why do you need this main1 function?

Comment (jsakhno) [reply homeworks/yaroslav_sakhno/hw04/tests.py]: @ProffesorRabbit I'm done

File: homeworks/yaroslav_sakhno/hw04/tests.py (hunk 1 above)
Comment (jsakhno) [reply homeworks/yaroslav_sakhno/hw04/tests.py]: @extsoft I did it

**Pull request merged**